Examples/GeoShape.md

```diff
@@ -0,0 +1,123 @@
+# GeoShape Fields Usage In RediSearch
+
+As of RediSearch 2.8.4, advanced GEO querying with GEOSHAPE fields is supported.
+
+Any object/library producing a
+[well-known text (WKT)](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) in `string` format can be used.
+
+In this example, we'll demonstrate how to use GeoShape fields in RediSearch with [NetTopologySuite](https://github.com/NetTopologySuite/NetTopologySuite) library.
```

Comment:
`Using GeoShape fields in searches with the [NetTopologySuite](https://github.com/NetTopologySuite/NetTopologySuite) library.`